src/components/dashboard/sidebar/offers/data/tests/reducer.test.js

```diff
@@ -31,12 +32,14 @@ describe('offers reducer', () => {
     const expected = {
       loading: false,
       offers: ['some data'],
+      offersCount: 4,
```

Comment:
Is there a reason why this is 4 here? Or do you just need a placeholder value?

Reply:
yeah, it's just a placeholder value